bio_embeddings/extract/pipeline.py

```diff
@@ -434,6 +473,12 @@ def predict_annotations_using_basic_models(model: str, **kwargs) -> Dict[str, An
     write_fasta_file(DSSP8_sequences, DSSP8_predictions_file_path)
     write_fasta_file(disorder_sequences, disorder_predictions_file_path)
 
+    if 'get_activations' in kwargs and kwargs['get_activations']:
+        # create files with activations for each multiclass prediction
+        concatenate_dataframe(DSSP3_raw).to_csv(DSSP3_raw_predictions_file_path, index=False)
+        concatenate_dataframe(DSSP8_raw).to_csv(DSSP8_raw_predictions_file_path, index=False)
+        concatenate_dataframe(disorder_raw).to_csv(disorder_raw_predictions_file_path, index=False)
```

Comment:
We generally use the id as index, this should write the same format as we use elsewhere:

```suggestion
        concatenate_dataframe(DSSP3_raw).set_index('seqID').rename_axis(None).to_csv(DSSP3_raw_predictions_file_path)
        concatenate_dataframe(DSSP8_raw).set_index('seqID').rename_axis(None).to_csv(DSSP8_raw_predictions_file_path)
        concatenate_dataframe(disorder_raw).set_index('seqID').rename_axis(None).to_csv(disorder_raw_predictions_file_path)
```

Reply:
Fixed